packages/compass-query-bar/src/components/query-bar.tsx

```diff
@@ -182,7 +182,6 @@ export const QueryBar: React.FunctionComponent<QueryBarProps> = ({
   onHideAIInputClick,
 }) => {
   const darkMode = useDarkMode();
-  const newExplainPlan = usePreference('newExplainPlan', React);
```

Comment:
Can probably remove this completely from preferences too

Reply:
Just kinda weird to have flags in the preferences package that don't affect anything, but yeah, FWIW the branching code is the part that causes more maintenance overhead, not the flags thenselves, so no strong opinion here on whether to keep them or not